packages/database/src/server/getDbMigrator.js

```diff
@@ -51,8 +93,26 @@ export const getDbMigrator = (forCli = false) =>
         },
       },
       cmdOptions: {
-        'migrations-dir': path.join(__dirname, '../core/migrations'),
+        'migrations-dir': SERVER_MIGRATION_DIR,
       },
     },
     forCli ? cliCallback : appCallback,
   );
+
+  // No need to exclude non-server migrations if we're creating a new migration
+  if (!process.argv.includes('create')) {
+    // 'core/migrations' folder is shared between server and browser
+    // We need to exclude non-server migrations before they are run
+    // ie: excluding migrations that have _meta.targets.includes('browser')
+    // This hook is called BEFORE the migrations are run,
+    // so we temporarily remove non-server migrations before they are run
+    instance.registerAPIHook(() => {
+      removeDirectoryIfExists(SERVER_MIGRATION_DIR);
+      createDirectory(SERVER_MIGRATION_DIR);
+      copyDirectory(MIGRATIONS_DIR, SERVER_MIGRATION_DIR);
+      removeNonServerMigrations();
```

Comment:
In the old server migrations process, do we now remove browser-only migrations (if we expect such a thing to exist)?

Reply:
Oh yeah sorry, it's remove _non_ server migrations, not remove server migrations. So that's exactly what this does.